packages/coinjoin/src/backend/CoinjoinMempoolController.ts

```diff
@@ -1,40 +1,57 @@
-import { isNotUndefined, promiseAllSequence } from '@trezor/utils';
-
 import type { BlockbookTransaction, MempoolClient } from '../types/backend';
 import { doesTxContainAddress } from './backendUtils';
 
+type MempoolStatus = 'stopped' | 'running';
+
 export type MempoolController = {
+    get status(): MempoolStatus;
+    start(): Promise<void>;
     update(): Promise<void>;
     getTransactions(addresses: string[]): BlockbookTransaction[];
 };
 
 export class CoinjoinMempoolController implements MempoolController {
     private readonly client;
-    private mempool: { [txid: string]: BlockbookTransaction };
+    private readonly mempool;
+    private readonly onTx;
+    private state: MempoolStatus;
```

Comment:
:+1: 
I did it because it's a backing variable for `get status()` property and dangling underscores were disabled. As an alternative, I propose to modify eslint rule in this file to allow underscores and use it like `private _status: MempoolStatus` and `get status() { return this._status }` which is somehow conventional.